docs/components.md

```diff
@@ -43,6 +43,28 @@ See also the more [detailed documentation for list components](lists.md).
 
 Shows a pretty checkbox. The same props are accepted as for `<input type="checked" />` elements. Use `value` for whether the checkbox is checked or not, rather than `checked`. If no `id` is passed, one will be generated and used.
 
+## ColumnWrapper
+
+This is a default column wrapper available to be used by the applications.  This will ensure all applications experience the same behavior for columns used in lists.
+
+Here is some sample code on usage:
+
+> ```
+> const LocationList = () => {
+> 	const scope = useSelector(getCurrentScope);
+> 	const name = "locationList" + scope;
+> 	const locations = unwrapImmutable(useSelector(mappedLocationListSelector));
+> 	useLoader(getLocations(scope), locationListLength);
+> 
+> 	return (
```

Comment:
The stuff above this line is probably not relevant for the example.

Reply:
Likewise the function closing, line 64 and 65, plus then you can remove a lot of whitespace.